debian/tests/control

```diff
@@ -2,9 +2,9 @@ Tests: growpart-loop
 Restrictions: needs-root isolation-machine allow-stderr
 Depends: cloud-guest-utils, fdisk, gdisk
 
-Tests: growpart-fsimage, growpart-fsimage-middle, growpart-start-matches-size
+Tests: growpart-fsimage, growpart-fsimage-middle, growpart-start-matches-size, growpart-fsimage-lvm
```

Comment:
This is not going to work as `growpart-fsimage-lvm` needs `needs-root isolation-machine`. The easiest thing to do is to add `growpart-fsimage-lvm` and the `lvm2` dependency to the section above, which already has the needed requirements. What I did test is a dedicated section:
```
Tests: growpart-fsimage-lvm
Restrictions: needs-root isolation-machine allow-stderr
Depends: cloud-guest-utils, fdisk, gdisk, lvm2
```
and the test passed.

Reply:
Thanks again for the thoughtful review.

The first thing I'd like to do is rename 'growpart-fsimage-lvm' to 'growpart-lvm' to avoid the confusion that this is an 'fsimage' test.  Its not, its a loop test as 'growpart-loop' is.   Also rename the 'debian/tests/growpart-' to match.

Then, I there are 2 options:

  1. add growpart-lvm to the 'Tests: growpart-loop' line and add 'lvm2' to the Depends of that "stanza"
  2.  add another stanza like you suggest (with the new 'growpart-lvm'). test.

I think that '1' in reality ends up making the test harness just run faster as 'growpart-loop' and 'growpart-lvm' would run in the same vm.  That is OK, there should not be issue with that.

I did '1'.  If you prefer '2', which admittedly does have better granularity, I'm fine with that too.

Reply:
Both the options are perfectly good IMO. I re-ran the autopkgtests and everything works fine. Thanks!